package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "program-examples",
+  "version": "1.0.0",
+  "description": "### :crab: Rust. :snake: Python. :ice_cube: Solidity. :link: All on-chain.",
+  "main": "index.js",
```

Comment:
should this be the index file that is inside the `scripts/lib` file?

Reply:
I removed the `main` property as we don't need it here - we won't import the `program-examples` package.